### ISSUE TITLE
System.OutOfMemoryException

### ISSUE THREAD

**Issue by dokuzsekiz** [UziTech/NSF2SQL#1]
268000 document read, when I get the following error.

System.OutOfMemoryException

**Comment by MySChik** [comment]
> 268000 document read, when I get the following error.
> 
> System.OutOfMemoryException

Hi,

Anyone has any solution? sorry, I'm new in this.

**Comment by UziTech** [comment]
One solution could be to use temporary files instead of ram to load the data.

Any pull requests would be appreciated.

**Comment by jaishmathews** [comment]
It's not system RAM. I checked and confirmed that system RAM is available, but this 32 bit application is producing memory error for more than 200000 + documents in the loop. So this memory meant by internal to application. We can't convert this to 64 bit based as of COM dependency I hope.

**Comment by jaishmathews** [comment]
> One solution could be to use temporary files instead of ram to load the data.
> 
> Any pull requests would be appreciated.

You meant by using files in place of existing object Dictionary<string, Table>?

**Comment by UziTech** [comment]
Yes. Removing the need to keep everything in memory by saving to files or something could help when there are lots of documents.

**Comment by SalihArik** [comment]
Could anyone solve the System.OutOfMemoryException error?
